src/detector/implementations/abstract_marker_interface.py

```diff
@@ -0,0 +1,27 @@
+import abc
+import datetime
+from src.common import \
+    EmptyResponse, \
+    ErrorResponse
+from src.common.structures import MarkerCornerImagePoint
+from src.common.structures.marker_status import MarkerStatus
```

Comment:
You should be able to import `MarkerStatus` from `src.common.structures`. If not, check the `__init__.py` in that folder